### PR TITLE
Removed all "unused variable" warnings.

### PR DESCRIPTION
**Pull request by sobolevn**
Nothing more than `_` before unused variables.

But there were some unused expression results. Do you even need these expressions? Tests were passing without them. If you really don't need them, I will remove them and update PR.

Thanks.

### CODE REVIEW

File: lib/mix/tasks/g.ex [hunk 1]
```diff
@@ -13,7 +13,7 @@ defmodule Mix.Tasks.Arc do
     def run([model_name]) do
       app_name = Mix.Project.config[:app]
       project_module_name = camelize(to_string(app_name))
-      model_destination = Path.join(System.cwd(), "/web/uploaders/#{underscore(model_name)}.ex")
+      _model_destination = Path.join(System.cwd(), "/web/uploaders/#{underscore(model_name)}.ex")
```
Comment (stavro) [inline lib/mix/tasks/g.ex]: Thanks! Can you remove this?

File: lib/arc/storage/s3.ex [hunk 2]
```diff
@@ -25,15 +25,15 @@ defmodule Arc.Storage.S3 do
   end
 
   def delete(definition, version, {file, scope}) do
-    destination_dir = definition.storage_dir(version, {file, scope})
+    _destination_dir = definition.storage_dir(version, {file, scope})
```
Comment (stavro) [inline lib/arc/storage/s3.ex]: Thanks! Can you remove this?

**Comment by stavro** [comment]
👍

**Comment by sobolevn** [comment]
Done.

**Comment by stavro** [comment]
Thanks!

**Pull request merged**